layouts/New/layouts.module.css

```diff
@@ -86,7 +86,7 @@
   .hexagonBackdrop {
     @apply absolute
       left-0
-      z-[-1]
+      -z-[1]
```

Comment:
Shouldn't it be just -z-1?

Reply:
In Tailwind `z-index` information can be multiples of 10 / or manually, I think there is no harm in saying 10 here. I will update it 👍